zipline/finance/transaction.py

```diff
@@ -36,6 +36,16 @@ def __init__(self, sid, amount, dt, price, order_id, commission=None):
     def __getitem__(self, name):
         return self.__dict__[name]
 
+    def __repr__(self):
```

Comment:
Very minor style point: this might be slightly nicer to read broken out over multiple lines:

```python
    def __repr__(self):
        template = "{cls}(sid={sid}, dt={dt}, amount={amount}, price={price})"
        return template.format(
            cls=type(self).__name__,
            sid=self.sid,
            dt=self.dt,
            amount=self.amount,
            price=self.price
        )
```